cmds/df/df.go

```diff
@@ -0,0 +1,153 @@
+//
```

Comment:
Would you mind adding our copyright header, like this one?

https://github.com/u-root/u-root/blob/master/cmds/ansi/ansi.go#L1

Reply:
will do - I'll put it in the next commit.